web/src/main/resources/META-INF/resources/webjars/common/webcert/components/wcSupportPanelManager/wcSrsPanelTab/wcSrsQuestionnaire.directive.html

```diff
@@ -0,0 +1,34 @@
+<div id="questionnaire" class="intyg-block__titlerow ng-scope">
+    <section>
+        <div >
+            <div ng-repeat="question in srs.questions track by $index" class="srs-question-element">
+                <b>{{question.text}}</b>
+                <div class="srs-question-answers srs-option-grid">
+                    <div class="srs-question-answer srs-option-grid-cell"
+                         ng-repeat="answer in question.answerOptions track by $index" for="{{answer}}">
+                        <wc-radio-wrapper>
+                            <input type="radio" ng-disabled="srs.consent !== 'JA' || srs.isForlangning" ng-change="change()"
+                                   id="q{{$parent.$parent.$index}}-a{{$parent.$index}}"
+                                   name="q{{$parent.$parent.$index}}" ng-model="question.model" ng-value="answer" />
+                            <label for="q{{$parent.$parent.$index}}-a{{$parent.$index}}"
+                                   ng-attribute-for="q{{$parent.$parent.$index}}-a{{$parent.$index}}">{{answer.text}}</label>
```

Comment:
Onödigt att ha både for och ng-attribute-for?  (Att använda $parent.$parent känns inte helt klockrent, men det kanske är svårt att härleda ett annat unikt id?

Reply:
Den där var lite konstig men 
$parent.$parent.$index är tänkt att peka på index för iterationen över frågor som görs högst upp, så det är i samma fil/komponent. 
Jag vet inte riktigt varför det blev ett "parent scope" men det verkar som att varje ng-repeat skapade sitt eget scope.

Reply:
Tar bort ng-attribute-for också